formatter.go

```diff
@@ -46,3 +46,8 @@ func RFC5424Formatter(p Priority, hostname, tag, content string) string {
 		p, 1, timestamp, hostname, appName, pid, tag, content)
 	return msg
 }
+func LoggerFormatter(p Priority, hostname, tag, content string) string {
+    msg := fmt.Sprintf("<%d>%s: %s",
```

Comment:
I'm not sure this does what it needs to.

```
$ logger -p local1.crit -t "cray" "ooooh noooo" 
Nov 9 19:08:19 smpp-05a cray: ooooh noooo
```

I have to assume we're running on different OSes, because I'm on OS X and this what I get:

```
$ logger -is -p local1.crit -t "cray" "ohh noo"
Nov  9 15:44:14  cray[30086] <Critical>: ohh noo
```

So, for one thing, you're going to want this to be:

```go
fmt.Sprintf("%s  %s[%d] <%s>: %s",
    time.Now().Format(time.Stamp),
    tag,
    os.Getpid(),
    p, // although we may need to convert it to a string like "Critical"?
    content)
```

And it needs test coverage.

But since this appears to differ across platforms, a name like `LoggerFormatter` seems too general to me.